<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskConfiguration</class>
 <widget class="QDialog" name="TaskConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Configuration</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QWidget" name="sidebar_container" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="sidebar_layout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="add_new_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Add New</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="add_button_layout">
        <item>
         <widget class="QPushButton" name="alarm_button">
          <property name="text">
           <string>Alarm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="region_button">
          <property name="text">
           <string>Region</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="line_button">
          <property name="text">
           <string>Line</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="current_tasks_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Current Tasks</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ZoneList" name="zone_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>45</number>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>0</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">3</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="video_container" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="video_layout">
      <item>
       <widget class="QLabel" name="stream_name_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">[Stream Name]</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="VideoTaskConfig" name="video_task_config">
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="op_button_layout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="confirm_op_button">
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancel_op_button">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="instruction_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignRight">
         <widget class="QDialogButtonBox" name="dialog_button_box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoTaskConfig</class>
   <extends>QGraphicsView</extends>
   <header>brainframe_qt.ui.dialogs.task_configuration.video_task_config.video_task_config</header>
  </customwidget>
  <customwidget>
   <class>ZoneList</class>
   <extends>QTreeWidget</extends>
   <header>brainframe_qt.ui.dialogs.task_configuration.zone_list.zone_list</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>alarm_button</sender>
   <signal>clicked()</signal>
   <receiver>TaskConfiguration</receiver>
   <slot>new_alarm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm_op_button</sender>
   <signal>clicked()</signal>
   <receiver>TaskConfiguration</receiver>
   <slot>confirm_zone_edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_task_config</sender>
   <signal>polygon_is_valid_signal(bool)</signal>
   <receiver>TaskConfiguration</receiver>
   <slot>enable_confirm_op_button(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_button_box</sender>
   <signal>accepted()</signal>
   <receiver>TaskConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_button_box</sender>
   <signal>rejected()</signal>
   <receiver>TaskConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>new_alarm()</slot>
  <slot>confirm_zone_edit()</slot>
  <slot>enable_confirm_op_button(bool)</slot>
 </slots>
</ui>
